great_expectations/expectations/metrics/column_aggregate_metrics/column_distinct_values.py

```diff
@@ -125,17 +157,10 @@ def _get_evaluation_dependencies(
             execution_engine=execution_engine,
             runtime_configuration=runtime_configuration,
         )
-
-        if isinstance(
-            execution_engine, (SqlAlchemyExecutionEngine, SparkDFExecutionEngine)
-        ):
-            dependencies["column.value_counts"] = MetricConfiguration(
-                metric_name="column.value_counts",
+        if metric.metric_name == "column.distinct_values.count.under_threshold":
```

Comment:
Does this mean this is the only column distinct values metric that has a dependency?

Reply:
Yes, that is correct.